include/vara/Feature/Feature.h

```diff
@@ -92,9 +92,7 @@ class Feature : public FeatureTreeNode {
   // Locations
   [[nodiscard]] bool hasLocations() const { return !Locations.empty(); }
 
-  void addLocation(FeatureSourceRange &Fsr) {
-    Locations.push_back(std::move(Fsr));
-  }
+  void addLocation(FeatureSourceRange Fsr) { Locations.push_back(Fsr); }
```

Comment:
```suggestion
  void addLocation(FeatureSourceRange Fsr) { Locations.push_back(std::move(Fsr)); }
```
For the value type FSR we need a move. Only before with the ref it was not needed